src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -904,6 +907,12 @@ def get(self, lv_name=None, vg_name=None, lv_path=None, lv_uuid=None, lv_tags=No
             raise MultipleLVsError(lv_name, lv_path)
         return lvs[0]
 
+    def dcopy(self):
```

Comment:
why is this named `dcopy` ? what is the reasoning? I understand that `self.copy` is already taken because it is inheriting from `list`, but I think it would be fine to override in this case as it is doing the same but with the LV elements. @andrewschoen what do you think about that? Does that approach seem reasonable to you?

Reply:
> why is this named dcopy ? 

it means deep copy. athough, i'm not sure now if it really is _deep_ copy. will check that tomorrow.

Reply:
Confirmed quickly; it is -

```
(Pdb) p type(lvs_copy)
<class 'ceph_volume.api.lvm.Volumes'>
(Pdb) p type(lvs)
<class 'ceph_volume.api.lvm.Volumes'>
(Pdb) del lvs_copy[0]
(Pdb) p lvs_copy
[</dev/fedora_localhost-live/root>, </dev/fedora_localhost-live/swap>]
(Pdb) p lvs
[</dev/ceph-e8b55c6a-a4a6-4b10-98a1-c78e37e53700/osd-block-e9373d71-918b-481a-b576-cdcb0b4a0fbd>, </dev/fedora_localhost-live/root>, </dev/fedora_localhost-live/swap>]
(Pdb) 
```

Reply:
That doesn't look like it is a deep copy... from the docs:

> A deep copy constructs a new compound object and then, recursively, inserts copies into it of the objects found in the original.

There is no recursive action here, and that behaves exactly the same as `.copy()`:

```
In [1]: foo = [1,2,3]                                                                                                                                                                                              

In [2]: bar = foo.copy()                                                                                                                                                                                           

In [3]: del foo[0]                                                                                                                                                                                                 

In [4]: foo                                                                                                                                                                                                        
Out[4]: [2, 3]

In [5]: bar                                                                                                                                                                                                        
Out[5]: [1, 2, 3]
```

In any case, if you really want to continue with the deep copy naming, could you call it `deepcopy` instead?